src/RadioGroup/RadioItem.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react'
+import { useUniqueId } from '../utils/id'
+import { RadioElement } from './RadioElement'
+import styled from 'styled-components'
+import { theme } from '../theme'
```

Comment:
Does S'mores follow the pattern of separating imports according to external / internal components / utils? It's been a while since I've worked on this repo so can't remember

Reply:
I think we do, I always forgot to follow what we agreed on tbh
I'll make an update